ETC.tsv

```diff
@@ -97,109 +97,109 @@ ETC_20150317_000096	$Although their bright color makes them pretty visible, it's
+ETC_20150317_000156	$Although slimes have similar appearance, they vary in species due to difference between the materials they originated from.
 ETC_20150317_000157	$Rusrat
-ETC_20150317_000158	During the times of Zachariel the Great, these monsters were placed to stop intruders. Its name stems from grave robbers who encountered it.
+ETC_20150317_000158	$Starting from the Royal Mausoleum of Zachariel the Great, these guardians were placed in many ruins to stop intruders. Its name stems from grave robbers who encountered it.
 ETC_20150317_000159	$Madakia
-ETC_20150317_000160	Court sorcerers created the Madakia to fend off intruding grave robbers.
+ETC_20150317_000160	$Court wizards created the Madakia to fend off intruders in case they are not mere grave robbers.
 ETC_20150317_000161	$Wheelen
-ETC_20150317_000162	In order to watch over sealed ruins that lacked an overseer, monsters were created with a diversity of magic. The wheelen is one of these monsters.
+ETC_20150317_000162	$The Wheelen is one of the magical creations made to protect sealed ruins without an overseer.
 ETC_20150317_000163	$Wheelen 3
 ETC_20150317_000164	$Mauros
-ETC_20150317_000165	The stone Calvary of the Royal Mausoleum are magically assembled to withstand even the heavily armed knight.
+ETC_20150317_000165	$The stone calvary of the Royal Mausoleum are the magical creations to withstand even the heavily armed knight.
 ETC_20150317_000166	$Truffle
-ETC_20150317_000167	Though the reason is unknown, this monster has an extreme hatred towards humans. It loves to attack them for food and survival.
+ETC_20150317_000167	$Though the reason is unknown, this monster has an extreme hatred towards humans. It prefer attacking them to hunting its prey or pursuing its survival.
 ETC_20150317_000168	$Treepellia
 ETC_20150317_000169	$Bramble
 ETC_20150317_000170	Bramble Ego
 ETC_20150317_000171	$Willow
 ETC_20150317_000172	$Willowna
 ETC_20150317_000173	$Hanaming
-ETC_20150317_000174	Fallen Hanaming leaves usually don't wither rapidly.
+ETC_20150317_000174	$Leaves fallen from a dead Hanaming wither rapidly.
 ETC_20150317_000175	$Chupacabra
-ETC_20150317_000176	Their numbers grew after the Great Plant Cataclysm; now they tend to compete with humans for territory.
+ETC_20150317_000176	$Their numbers grew after the Great Plant Cataclysm; now they tend to compete with humans for territory.
 ETC_20150317_000177	$Gray Chupacabra
 ETC_20150317_000178	$Large Gray Chupacabra
-ETC_20150317_000179	Chupacabras move secretly. Based on this, it is more dangerous for humans if there are elite Chupacabras.
+ETC_20150317_000179	$Chupacabras move secretly. Based on this, it is more dangerous for humans if there are elite Chupacabras.
 ETC_20150317_000180	$Yellow Chupacabra
 ETC_20150317_000181	$Popolion
-ETC_20150317_000182	Some say a Popolion's voice can even break glass bottles.
+ETC_20150317_000182	$Some say a Popolion's voice can even break glass bottles.
```

Comment:
"A Popolion's voice is said to break even glass bottles."

Popolion is the subject and "even" moved to after "break" to put more emphasis on the power of its voice.